opl/opl_sdl.c

```diff
@@ -154,6 +155,60 @@ static void AdvanceTime(unsigned int nsamples)
     SDL_UnlockMutex(callback_queue_mutex);
 }
 
+static void MixAudioFormat(Uint8 *dst, const Uint8 *src, Uint32 len)
+{
+    Sint16 src1, src2;
+    int dst_sample;
+    const int gain = 8;
```

Comment:
This is a copy of the SDL function, see: https://github.com/libsdl-org/SDL/blob/39eab4bf44472790dddf02490ac260f810c30c9a/src/audio/SDL_mixer.c#L85

Reply:
Since `AUDIO_S16SYS` and `AUDIO_S16LSB`/`AUDIO_S16MSB` are defined as macros, you could let the preprocessor decide about the code path.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/829/commits/ba78acb54e81c3c3f1ea06a11f62e553259dd25e